pandas/tests/io/parser/test_common.py

```diff
@@ -2241,3 +2241,19 @@ def test_read_table_delim_whitespace_non_default_sep(all_parsers, delimiter):
 
     with pytest.raises(ValueError, match=msg):
         parser.read_table(f, delim_whitespace=True, delimiter=delimiter)
+
+
+def test_dict_keys_as_names(all_parsers):
+    # GH: 36928
+    data = "1,2"
+
+    keys = {"a": int, "b": int}.keys()
+    parser = all_parsers
+
+    result = parser.read_csv(StringIO(data), names=keys)
+    expected = DataFrame({"a": [1], "b": [2]})
+    tm.assert_frame_equal(result, expected)
+        parser.read_csv(f, delim_whitespace=True, sep=delimiter)
+
+    with pytest.raises(ValueError, match=msg):
+        parser.read_csv(f, delim_whitespace=True, delimiter=delimiter)
```

Comment:
Can you remove this? Probably why CI is failing.

Reply:
I had to add that to fix the merge conflict with master...

Reply:
Though maybe I shouldn't have touched it.

Reply:
You can still delete it, only have to delete the code below L2255 and commit (being sure to keep the EOF newline else CI will fail)

Reply:
Delete until EOF? My test function is below L2255 and there is another test function before it.

Reply:
Oh, just saw it - no idea how that got there... Hopefully it's fixed now. Sorry about the confusion.